recipes/gazebo/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "gazebo" %}
+{% set version = "9.11.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  - url: https://bitbucket.org/osrf/{{ name }}/get/{{ name }}{{ version[0] }}_{{ version }}.tar.gz
+    sha256: df25536768f457e30e5213ba7b490d0e3b87c69ab5f55f9142179fad0cbd62d4
+    patches:
+      - use-external-libs-config.patch
+      - qwt-fix.patch
+
+build:
+  number: 0
+  skip: true  # [not win or vc<14]
+  run_exports:
+    - {{ pin_subpackage('gazebo', max_pin='x.x') }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}              # [not win]
+    - {{ compiler('c') }}                # [not win]
+    - vs2017_win-64                      # [win64]
+    - vs2017_win-32                      # [win32]
```

Comment:
the ``vs`` here is not necessary, as you are already skipping ``vc<14`` just the ``vs2017`` will be used